mirdata/msd.py

```diff
@@ -0,0 +1,373 @@
+# -*- coding: utf-8 -*-
+"""Million Song Dataset Loader
+
+The Million Song Dataset is a freely-available collection of audio features and metadata for a million contemporary popular music tracks.
+
+More details can be found on http://millionsongdataset.com/
+
+Attributes:
+    DATASET_DIR (str): The directory name for Million Song Dataset. Set to `'msd'`.
+
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tables
+
+import mirdata.utils as utils
+import mirdata.download_utils as download_utils
+
+DATASET_DIR = "MSD"
+
+
+DATA = utils.LargeData("msd_index.json")
+
+
+class Track(object):
+    """MSD Track class
+
+    Args:
+        track_id (str): Track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets`
+
+    Attributes:
+        Please refer to http://millionsongdataset.com/
```

Comment:
Can you document the top level attributes? `analysis`, `metadata`, etc?

Reply:
Yep, will do!